components/board/genericlinux/board.go

```diff
@@ -533,7 +519,8 @@ func (b *sysfsBoard) GPIOPinByName(pinName string) (board.GPIOPin, error) {
 }
 
 func (b *sysfsBoard) periphGPIOPinByName(pinName string) (board.GPIOPin, error) {
-	pin, hwPWMSupported, err := b.getGPIOLine(pinName)
+	pin, err := b.getGPIOLine(pinName)
+	hwPWMSupported := false
```

Comment:
If this value is always false, I suspect we can get rid of it as a field in `periphGpioPin`, too, which will simplify that class further. but this PR is already pretty large, and I don't have a sense of whether that change should go here because it's on the same topic, or separately because this PR is already big. In hindsight, I wish we'd had one PR to remove the extra parts of the jetson data.go, and a second PR for all the other changes.

Reply:
i can fix this later in another pr